spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -13,15 +15,15 @@
   it 'finds context without `when` at the beginning' do
     expect_offense(<<-RUBY)
       context 'the display name not present' do
-              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Start context description with 'when', or 'with'.
+              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Context description should match /^when\\b/, or /^with\\b/.
```

Comment:
with love to see an example with 3 patterns

Reply:
I updated it
https://github.com/rubocop/rubocop-rspec/blob/c017ffd20d2b78f7fda8b129d1aa85ad80edac70/spec/rubocop/cop/rspec/context_wording_spec.rb#L15-L21

Reply:
Thank you